CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [2.0.0](https://github.com/deshaw/jupyterlab-limit-output/compare/v1.0.1...v2.0.0) (2023-04-04)
```

Comment:
For release date `(UNRELEASED)` is correct until we choose to release. I suspect it doesn't make sense to release before lab is finalized.